### DISCUSSION TITLE
Small object detection - what is the minimum bounding box size?

### DISCUSSION THREAD

**Discussion by DSA101** [ultralytics/yolov5#5795] [Q&A] [answered]
What is the smallest bounding box size for Yolo5 training? I'll need to detect medium and small objects (think cars at distance or ships at distance) that can be just a couple of pixels in size. It'll be just one class of objects that I'll need to detect.

**Comment by glenn-jocher** [comment, marked as answer]
@DSA101 default label candidate criteria includes >2 pixels. This can be manually modified here:
https://github.com/ultralytics/yolov5/blob/53349dac8e9fb447bb43319811699bb72d1c2470/utils/augmentations.py#L272-L277
